.pipelines/.vsts-vhd-builder.yaml

```diff
@@ -32,10 +32,9 @@ variables:
   value: 'mcr.microsoft.com/oss/azcu/go-dev:v1.38.1'
 
 stages:
-  - stage: build_vhd
-    dependsOn: []
+  - stage: build
     jobs:
-    - job: build
+    - job: build-2204-gen2-containerd
```

Comment:
for my own knowledge, what are we building those for ?  from the looks of it , those are not all of the BHD images we built, since it only 3 of them

Reply:
I'm not 100% sure. But I think it's a small setup to run on each PR. Probably with a goal to consume less resources, be faster and be less flaky.

I think it's the main place where we want to run our e2e. Before people merge PRs.

Reply:
ok I see, so a subset of the builds during the PR to get a somewhat good coverage while focussing on efficiency.